src/main/resources/assets/blast/lang/es_es.json

```diff
@@ -0,0 +1,66 @@
+{
+  "item.blast.bombard": "Cañón",
+  "item.blast.slime_bombard": "Cañón de slime",
+  "item.blast.amethyst_bombard": "Cañón de amatista",
+  "item.blast.frost_bombard": "Cañón de escarcha",
+  "item.blast.bomb": "Bomba",
+  "item.blast.trigger_bomb": "Activador de la bomba",
```

Comment:
Wouldn't this mean "activator of the bomb" rather than "bomb that can be activated" ?

Reply:
Sorry, I didn't understand the context when writing that. I thought that's what it meant by "trigger bomb". "Bomba activable" or "bomb que puede estar activado" (which is activatable bomb) would be the correct translation?